react/PushClientButton/index.jsx

```diff
@@ -0,0 +1,38 @@
+import styles from './styles.styl'
+
+import React from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+
+import DeviceIcon from '../../assets/icons/illus/device-laptop.svg'
+
+const ButtonClient = props => {
+  const { label, href, onClick, className } = props
+  return (
+    <a
+      href={href}
+      //eslint-disable-next-line react/jsx-no-target-blank
+      target="_blank"
+      className={cx(styles['c-btn-client'], className)}
+      onClick={onClick}
+    >
+      <figure>
+        <img width="32" height="32" src={DeviceIcon} />
```

Comment:
I'm wondering if `DeviceIcon` should not be a props with this default value. It'll be easier to override?

Reply:
hum could be yes. We don't have any other use of this banner but I guess it would be future proof.